docs/block-processing/us.md

```diff
@@ -152,23 +168,27 @@ So if a value is going to be updated then a field is `Just` and `Nothing` otherw
 
 ### Adoption of block version
 
-Informally, `BlockVersion` is **adopted** if sum of block issuers' stakes, 
-which issued blocks of this `BlockVersion` at least once, takes a significant part of the stake.
+Informally, `BlockVersion = (Maj, Min, Alt)` is **adopted** iff:
```

Comment:
This and following three sections should be unified into one and rewritten in consistent way